armi/reactor/tests/test_blocks.py

```diff
@@ -16,30 +16,21 @@
 # pylint: disable=missing-function-docstring,missing-class-docstring,abstract-method,protected-access,no-member,invalid-name,consider-using-f-string
 import copy
 import math
-import os
-import unittest
 import numpy
 from numpy.testing import assert_allclose
+import os
```

Comment:
I believe these imports were moved in error.  Have you seen our docs on the ordering of `import` statements?

https://terrapower.github.io/armi/developer/standards_and_practices.html#import-ordering

Reply:
Completed.